changelog.txt

```diff
@@ -1,3 +1,30 @@
+14.0 (2024-04-13)
+------------------------------------------------------------------------
+Update: New title game for 14.0
+Fix #12477: Use std::filesystem::rename instead of Windows Shell API call (#12478)
```

Comment:
```suggestion
Fix #12477: Crash when launching OpenTTD from within a Dropbox folder (#12478)
```

The current text is a technical description of the change. Not what it changed or why :)